uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.js

```diff
@@ -216,6 +216,7 @@ define([
 
         doc.activity_date_time = $scope.parseDate(doc.activity_date_time) || new Date();
         doc.expire_date = $scope.parseDate(doc.expire_date);
+        doc.status_id = (!$scope.isRole('admin')) ? '2' : $scope.document.status_id; // 2 => 'Awaiting Approval'
```

Comment:
> Why is the status id hardcoded?
As the status id of `Awaiting Approval` is always `2`, not not going to change, so, it is hard coded at the moment.

> So if i'm staff or manager the id is always going to be "Awaiting approval"?
Not always, if the document is updated with attachment the the status will be `Awaiting Approval`. But if, attachment are not attached the status of the document will be `Awaiting Upload`.